config/initializers/local_config.rb.example

```diff
@@ -37,6 +37,7 @@ ActionMailer::Base.smtp_settings = {
 # ** FrontlineCloud **
 # configatron.default_settings.outgoing_sms_adapter = "FrontlineCloud"
 # configatron.default_settings.frontlinecloud_api_key = "xxx"
+# configatron.universal_sms_token = "50b488c71be4e9cd0ba34a7bf0b7053f"
```

Comment:
This looks like it's part of the frontline cloud settings. Perhaps put in a separate paragraph and add a comment?